timesketch/ui/static/components/explore/explore-event.html

```diff
@@ -0,0 +1,91 @@
+<!--
+Copyright 2015 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<table id="search-result-table" width="100%">
+    <tr>
+        <td ts-timeline-color timeline-colors="meta.timeline_colors" index-id="event._index" width="210px">
+            {{::event._source.datetime}}
+        </td>
+
+        <td width="35px;" >
+            <i ng-click="star = !star;toggleStar()" ng-class="{true: 'fa fa-lg fa-star icon-yellow', false: 'fa fa-lg fa-star-o icon-grey'}[star]"></i>
+        </td>
+
+        <td class="event-message" ng-click="getDetail();showDetails = !showDetails">
+            <span ng-show="comment">
+                <i class="fa fa-comments"></i>
+            </span>
+            [{{::event._source.timestamp_desc}}] {{::event._source.message|limitTo:500}}
```

Comment:
The principle is to not use normal bindings where it is not necessary :) The comments in the event directive are now fixed. Thanks for pointing this out!